app/policies/work_policy.rb

```diff
@@ -3,12 +3,19 @@
 
 # Authorization policy for Work objects
 class WorkPolicy < ApplicationPolicy
-  sig { returns(T::Boolean) }
+  alias_rule :edit?, to: :update?
 
   # Only depositors in a specific collection are able to create new collection members
+  sig { returns(T::Boolean) }
   def create?
     collection = record.collection
     collection.depositor_ids.include?(user.id) ||
       collection.managers.include?(user.email.delete_suffix('@stanford.edu'))
   end
+
+  # Only the depositor may edit/update a work
+  sig { returns(T::Boolean) }
+  def update?
+    record.depositor == user
```

Comment:
Do we have a use case for collections managers to be able to edit/update works deposited by other users in their collection? @amyehodge (Not saying we need to have that in this PR, but it would be easy to add?)

Reply:
Definitely. This is a must-have.

Reply:
I would say that is a question for a new ticket.

Reply:
https://github.com/sul-dlss/happy-heron/issues/363